File: Questões/Linguagem C/1020.c
#include <stdio.h>
 
int main() {
 
    int days, months, years;
    months = 0;
    years = 0;

    scanf("%d", &days);

    while (days >= 365){
        days -= 365;
        years++;
    }
    while (days >= 30){
        days -= 30;
        months++;
    }

    printf("%d ano(s)\n", years);
    printf("%d mes(es)\n", months);
    printf("%d dia(s)\n", days);
 
    return 0;
}

File: codes/1000/Python/1020.py
days = int(input())
months = 0
years = 0

while days >= 365:
    years += 1
    days -= 365

while days >= 30:
    months += 1
    days -= 30

print(f'{years} ano(s)\n{months} mes(es)\n{days} dia(s)')
